sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -2785,6 +2785,17 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase with ExecutionE
     Map.empty
   )
 
+  def invalidFileExtensionError(functionName: String, fileExtension: String): RuntimeException = {
+    new SparkIllegalArgumentException(
+      errorClass = "INVALID_PARAMETER_VALUE.FILE_EXTENSION",
+      messageParameters = Map(
+        "functionName" -> toSQLId(functionName),
+        "parameter" -> toSQLId("fileExtension"),
+        "fileExtension" -> toSQLId(fileExtension),
+        "acceptable" -> "File extension is limited to exactly 3 letters (e.g. csv, tsv, etc...)"))
+  }
+
+
```

Comment:
Could you remove this extra empty line?

Reply:
The line has been removed.

Reply:
I also changed "fileExtension" to extension.